acnportal/signals/tariffs/tariff_schedules/pge_a10_tou_aug_2019.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Just wondering, are these files copied directly from some database? i.e. are they guaranteed to be correct?

Reply:
Nope. Currently hand coded by your's truly. It would be great if we could pull these in programmatically from the Utility Rates database. However, this is a tall order from what I've seen from looking at their API. It seems like some utilities have some pretty complex formulas for tariffs that aren't easily expressed in a standard form.

Reply:
Maybe we could support the simple ones and throw an error for the more complex ones.